src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -31,6 +31,8 @@ namespace System.Collections
 #if FEATURE_NETCORE
     [FriendAccessAllowed]
 #endif
+    /// <summary>Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is dynamically increased as required.To browse the .NET Framework source code for this type, see the Reference Source.</summary>
```

Comment:
@stephentoub @cartermp we don't want remarks and other parts of the documentation here. First, they are not part of the IntelliSense files. Second, the plan is to add that kind of content via Markdown files.

That last sentence about browsing is actually in the summary for a lot of the APIs. Let me try to change the tool to prevent that from making it into the source.

For formatting the comments, I can investigate if it can be done before the code formatter too. What would be the desirable max length here? And do you only want to break into multiple lines if it goes past that length or do you always want to do that?